review_test.rb

```diff
@@ -0,0 +1,105 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './employee.rb'
+require './department.rb'
+
+class ReviewTest <Minitest::Test
+
+  def test_employee_exists
+    assert Employee
+  end
+
+  def test_department_exists
+    assert Department
+  end
+
+  def test_employees_have_salary_and_name
+    sara = Employee.new("Sara Bupkis" , 100000)
+    assert_equal 100000, sara.salary
+    assert_equal "Sara Bupkis", sara.name
+  end
+
+  def test_adding_employees_to_deptartment
+    sara = Employee.new("Sara Bupkis" , 100000)
+    department = Department.new("Sales")
+    department.add(sara)
+    assert_equal "Sales", sara.department
+  end
+
+  def test_getting_department_salary_totals
+    sara = Employee.new("Sara Bupkis" , 100000)
+    dave = Employee.new("Dave Mathis", 300)
+    department = Department.new("Sales")
+    department.add(sara)
+    department.add(dave)
+    assert_equal 100300, department.salary
+  end
+
+  def test_reviewing_an_employee
+    dave = Employee.new("Dave Mathis",300)
+    review_text = "Dave is terrible, I hate Dave. If I had a gun with 2"+
+    "bullets and Hitler and Dave were standing infront of me I would shoot"+
+    " Dave twice. \nNot only is Dave terrible at his job we pay him far too much."+
+    " Dave makes almost a dollar a day. This is insanity, Dave should be paying us to work here."
+    dave.review = review_text
+    assert_equal review_text, dave.review
+  end
+
+  def test_mark_an_employee_satisfactoy_or_not
+    dave = Employee.new("Dave Mathis",300)
+    dave.good_employee = false
+    refute dave.good_employee
+  end
+
+  def test_giving_an_employee_a_raise
+    sara = Employee.new("Sara Bupkis" , 100001)
+    sara.raise_salary(50)
+    assert_equal 150001.50, sara.salary
+  end
+
+  def test_raise_department_salary
+    sara = Employee.new("Sara Bupkis" , 100000)
+    dave = Employee.new("Dave Mathis", 300)
+    department = Department.new("Sales")
+    department.add(sara)
+    department.add(dave)
+    department.department_raise(20000)
+    assert_equal 120300, department.salary
+  end
+
+  def test_raise_department_salary_only_if_employees_are_good
+    sara = Employee.new("Sara Bupkis" , 100000)
+    dave = Employee.new("Dave Mathis", 300)
+    department = Department.new("Sales")
+    department.add(sara)
+    department.add(dave)
+    dave.good_employee = false
+    department.department_raise(20000)
+    assert_equal 120300, department.salary
+    assert_equal 300, dave.salary
+  end
+
+  def test_hard_mode
```

Comment:
Cool.  I like that you have written a good test for hard mode even though you didn't get to it.  In the real-world, you only commit when tests pass, but I appreciate you showing me this.